x-pack/plugin/core/src/main/resources/logs-default-mappings.json

```diff
@@ -0,0 +1,152 @@
+{
```

Comment:
@ruflin would `logs-generic-mappings` be more aligned with Observability in terms of naming?

Reply:
I think it should be `logs-mappings` and leave out `generic` or `default` as it applies to all datasets.

Reply:
I went with `logs-default-mappings` to allow a user to create their own `logs-mappings` that was explicitly *not* the default one, I also figured it would have a lower chance of existing conflict with any templates a user may have installed